desktop-app/src/renderer/components/ToolBar/index.tsx

```diff
@@ -6,17 +6,40 @@ import {
   setRotate,
 } from 'renderer/store/features/renderer';
 import { Icon } from '@iconify/react';
+import { useEffect, useRef } from 'react';
 import NavigationControls from './NavigationControls';
 import Menu from './Menu';
 import Button from '../Button';
 import AddressBar from './AddressBar';
 import ColorSchemeToggle from './ColorSchemeToggle';
 
+function useKey(key: any, cb: any) {
```

Comment:
Use of `any` should be avoided as much as possible as it beats he purpose of ts. Any specific reason why you chose to make these `any`?

This can be `function useKey(key: string, cb: () => void)`

Reply:
No, There is no specific reason, I am just new with typescript😅.

Reply:
@codewithmitesh I'm planning to cut a release this week, can you resolve this so that we can merge this in for the release.